helm/klustercost/values.yaml

```diff
@@ -13,3 +13,8 @@ postgresql:
     size: 1Gi
     storageClass: "default"
   serviceType: ClusterIP
+
+monitor:
+  image: klustercost:1
+  resincTime: "300"
```

Comment:
We can keep resinc instead of resync for now since it also needs adjustments in go code. Will create issue for this.